package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "skeletor-core",
+  "version": "0.1.0",
+  "description": "The core engine for the Skeletor family of build tools",
+  "main": "index.js",
+  "scripts": {
+    "test": "jest"
```

Comment:
May want to add the linter in here. I pulled the rules out when I did work on objectUtils, so you can just copy that stuff over. It's just single script line.

Reply:
Ok, linting has been added. I'd rather not go into detail about how much time I spent removing trailing spaces from lines of code.

Reply:
Haha. In case you weren't aware, you can run `npm run lint -- --fix` and it should do most of the spacing stuff for you. Sometimes I just make a new command called `lint:fix` cause typing is lame.